keyboards/kyria/keymaps/tessachka/rules.mk

```
@@ -0,0 +1,3 @@
+OLED_DRIVER_ENABLE = yes   # Enables the use of OLED displays
+RGBLIGHT_ENABLE = yes      # Enable keyboard RGB underglow
+MOUSEKEY_ENABLE = YES
```

I think this does need to be lower case. 

```suggestion
MOUSEKEY_ENABLE = yes
```